### PR TITLE
rename files if copied in session as reference and correlate

### LINKED ISSUE #184: If the reference and correlate image has the same filenames then they overwrite each other when copied to the session folder
We need to rename the files when they get copied to the session folder, for all file types.

### PR DESCRIPTION
**Pull request by paskino**
closes #184

### CODE REVIEW

File: src/idvc/dvc_interface.py [hunk 1]
```diff
@@ -577,16 +577,19 @@ def SelectImage(self, image_var, image, label=None, next_button=None):
...
+                    new_file_name = 'correlate'
                 self.image_copied[image_var] = True
                 self.create_progress_window("Copying", "Copying files", 100, None)
                 self.progress_window.setValue(1)
                 for file_num, f in enumerate(files):
                     file_name = os.path.basename(f)
                     file_ext = file_name.split(".")[-1]
                     if file_ext == "mhd":
-                        new_file_dest = os.path.join(file_name[:-3] + "mha")
+                        new_file_dest = os.path.join(new_file_name + "mha")
```
Comment (lauramurgatroyd) [inline src/idvc/dvc_interface.py:590]:
```suggestion
                        new_file_dest = os.path.join(new_file_name + ".mha")
```

**Review by lauramurgatroyd — CHANGES_REQUESTED** [review]
I found this does not work for tiffs - I get the following error:
image

And on the command line it gives an error: "When reading multiple files, all files must TIFF formatted."

I believe what is happening is it copies every tiff file to have the same new name, so they all overwrite each other.

**Comment by paskino** [comment]
I think I fixed the behaviour, but see also #193

**Review by lauramurgatroyd — CHANGES_REQUESTED** [review]
I encountered a couple of issues:

- When I encountered an error when clicking "Visualise" [I think the error was due to trying to downsample a compressed .mha file? ]  I had the error message pop up, but the progress dialog remained in place.

- Issues with .mhd files,  as the .mha created is compressed so can't be read by the resample reader. [ see comment ]

**Comment by lauramurgatroyd** [comment]
Just fixed this issue:

> When I encountered an error when clicking "Visualise" [I think the error was due to trying to downsample a compressed .mha file? ] I had the error message pop up, but the progress dialog remained in place.

**Review by lauramurgatroyd — APPROVED** [review]

**Pull request merged**